mail/rspamd/Portfile

```diff
@@ -2,78 +2,98 @@
 
 PortSystem          1.0
 PortGroup           cmake 1.0
-PortGroup           github 1.0
 
-github.setup        vstakhov rspamd 0.8.3
-
-version             0.8.3
-revision            3
+name                rspamd
+version             1.9.2
+revision            0
 categories          mail
 license             BSD
 maintainers         nomaintainer
 platforms           darwin
 description         Rspamd filtering system is created as a replacement of popular spamassassin \
-                    spamd and is designed to be fast, modular and easily extendable system.
+    spamd and is designed to be fast, modular and easily extendable system.
 long_description    ${description}  Rspamd core is written in C language using event driven \
-                    paradigma. Plugins for rspamd can be written in lua. Rspamd is designed to \
-                    process connections completely asynchronous and do not block anywhere in code.
+    paradigma. Plugins for rspamd can be written in lua. Rspamd is designed to \
+    process connections completely asynchronous and do not block anywhere in code.
 homepage            http://rspamd.com
 
-if {${github.version} eq ${version}} {
-    master_sites        http://rspamd.com/downloads
-    use_xz              yes
-    checksums           rmd160  dc43a24c9eef555bd390ab31c4aa1deb8c5e7704 \
-                        sha256  20f788b08406fdaa8148d726e7ba18940108afff2d48b4f47726e3f2a6a88daf
-} else {
-    fetch.type          git
-    post-fetch {
-        system -W ${worksrcpath} "git submodule update --init"
-    }
-}
+platforms           darwin
+supported_archs     noarch
+license             Apache-2
 
-depends_build-append port:pkgconfig
+master_sites        https://github.com/rspamd/rspamd/archive
+distname            ${version}
+extract.suffix      .tar.gz
+worksrcdir          rspamd-${version}
 
+# wget https://github.com/rspamd/rspamd/archive/1.9.2.tar.gz
+# openssl dgst -rmd160 1.9.2.tar.gz
+# openssl dgst -sha256 1.9.2.tar.gz
+
+checksums           rmd160  0343495b976a97275e015256ddfc307b8ace176a \
+    sha256  69a39cb83a4d28da2fc2fc2dc7c8e07435557e50e19cfdf73bd1ebd964da80f9
+
+## Add ragel dependence after this issue resolved https://trac.macports.org/ticket/58321#comment:6
+## In the meantime, roll back to 6.10
+## git clone --single-branch https://github.com/macports/macports-ports.git
+## cd macports-ports/
+## git checkout 4b72f12e19e746d94db029eb72d78169e1bf7649
+## cd lang/ragel
+## sudo port install
+depends_build-append port:pkgconfig port:luajit port:ragel
 depends_lib         port:libevent \
```

Comment:
While you are at it, change `depends_lib` to `depends_lib-append` to not overwrite portgroup dependencies.